### DISCUSSION TITLE
Wanted help in achieveing these following 3 tweaks

### DISCUSSION THREAD

**Discussion by SkewelVsAll** [moudey/Shell#155] [Q&A]
1) I wanted to know how I can reduce the intensity of the shadows of the Shell right click menu to something close or similar to the stock context menu.

2) Wanted to know how I could set Shell as my default system-wide context menu.

3) How do I add my GPUs control panel (AMD Adrenalin Software in my case) in the Shell context menu?

Thanks

**Comment by moudey** [comment]
Open `shell.nss` file in **Shell** folder
1. To customize the size and color of the shadow
```
shell
{
    set
    {
        theme
        {
            shadow
            {
                enabled=1
                size=4 // value from 0 to 20
                offest = 2
                color=#000000
                opacity=25
            }
        }
    }
}
```

2. Shell does not support system-wide context menu yet
3. In `dynamic` section, enter this command
```
item(type='desktop' title='AMD Adrenalin Software' image cmd='path\to\amd.exe')
```

Save file, press `CTRL`+`RIGHT-CLICK` or restart Explorer to reload settings.

> To restart Explorer use this command `shell.exe -restart `

**Reply by moudey** [reply]
Shell only works with 3rd party that use system shell context menus

**Reply by SkewelVsAll** [reply]
Hey, sorry for the late response - so far I have only been able to add the AMD Adrenaline driver as an entry on Shell and I'm okay with it.

Another issue that I noticed is that - everything I have something copied in the clipboard the "Paste" entry on Shell seems to change position.

For example: If I have nothing copied the "Paste" option will be greyed out and it will be listed as the 4th option from the top everytime I right click the mouse. But if I do have something copied in my clipboard, the option shifts it's position to the being the bottom 3rd/4th option with other similar options such as "Paste shortcut" etc right below it. 

I wanted to know - is there any way, that I could permanently fix the position of the "Paste" option to being at the top of the context menu?

**Reply by moudey** [reply]
Use the `position` property to specify the position of the menuitem
```
static
{
    // find by name
    item(find='"paste"' pos=5)
    // or find by id
    item(where=this.id==id.paste pos=5)
}
```

**Reply by SkewelVsAll** [reply]
Oh, alright. Thanks.

Also, I was wondering would it be possible for you to replicate the menu items with their positions, seperation bar etc as seen on a fresh install of Windows 10 and ship it out by default with future versions of Shell, where the users job will only trickle down to adding custom options if they wish to? 

Because me and many other casual users just want the same consistent context menu options available like in Windows 10, with the difference being that Shell UI is of Windows 11.

Thanks.
